From 6769f44c20bea54e59a54403192c8f6f620d4b46 Mon Sep 17 00:00:00 2001
From: Hang Xiaoqian <hangxiaoqian@wxiat.com>
Date: Sun, 9 Oct 2022 17:01:25 +0800
Subject: [PATCH] sw64: fix compile errors when CONFIG_STACKTRACE is not set
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6769f44c20bea54e59a54403192c8f6f620d4b46
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56QAM

--------------------------------

The stacktrace.c should be always compiled.

Signed-off-by: Hang Xiaoqian <hangxiaoqian@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/kernel/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/sw_64/kernel/Makefile b/arch/sw_64/kernel/Makefile
index 02facabae2d9..850f6dfddd1f 100644
--- a/arch/sw_64/kernel/Makefile
+++ b/arch/sw_64/kernel/Makefile
@@ -17,10 +17,9 @@ obj-y    := entry.o fpu.o traps.o process.o sys_sw64.o irq.o \
 	    irq_sw64.o signal.o setup.o ptrace.o time.o \
 	    systbls.o dup_print.o tc.o timer.o \
 	    insn.o early_init.o topology.o cacheinfo.o \
-	    vdso.o vdso/ hmcall.o
+	    vdso.o vdso/ hmcall.o stacktrace.o
 
 obj-$(CONFIG_ACPI) += acpi.o
-obj-$(CONFIG_STACKTRACE) += stacktrace.o
 obj-$(CONFIG_SMP)	+= smp.o
 obj-$(CONFIG_PCI)	+= pci.o pci-sysfs.o
 obj-$(CONFIG_MODULES)	+= module.o
-- 
2.33.0

